packages/sw-precaching/src/lib/models/precache-entries/object-precache-entry.js

```diff
@@ -1,43 +1,43 @@
-import ErrorFactory from '../error-factory';
-import RevisionedCacheEntry from './revisioned-cache-entry.js';
+import ErrorFactory from '../../error-factory';
+import RevisionedCacheEntry from './base-precache-entry.js';
 
 class DefaultsCacheEntry extends RevisionedCacheEntry {
-  constructor({revisionID, revision, url, cacheBust}) {
+  constructor({entryID, revision, url, cacheBust}) {
```

Comment:
Could I convince you to move to the `assert` library for these checks?

I'm happy enough with explicitly using `ErrorFactory` for more complex error reporting, but `assert` was created and is used elsewhere for basic parameter validation.

In addition to being consistent with the other modules, using `assert` would make it easier to toggle on this sort of validation during development and disable it in a production deployment. (Assuming you think that's a good idea—chime in on https://github.com/GoogleChrome/sw-helpers/issues/93)